automatic_symbol_loading/automatic_symbol_loading.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
+from pathlib import Path
+import gdb
```

Comment:
You should add a blank line above `import gdb` to separate standard and 3rd-party imports.

Reply:
OK, added